services/order.py

```diff
@@ -0,0 +1,34 @@
+
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, User
+from services.movie_session import get_movie_session_by_id
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None,
+) -> None:
+    with transaction.atomic():
+        user = User.objects.get(username=username)
```

Comment:
The `get` method will raise a `DoesNotExist` exception if no `User` with the given `username` is found. Consider handling this exception or documenting it in the function's docstring to inform users of this behavior.